src/main/kotlin/org/jetbrains/research/transformations/Main.kt

```diff
@@ -1,5 +1,48 @@
 package org.jetbrains.research.transformations
 
+import com.github.gumtreediff.gen.python.PythonTreeGenerator
+import com.github.gumtreediff.tree.ITree
+import com.github.gumtreediff.tree.TreeContext
+import java.io.File
+import org.jetbrains.research.transformations.Transformations.*
+
+fun setupParser() {
+    //copy parser file into /tmp directory, required by Gumtree
+    val sourcePath :String = "src/main/resources/pythonparser"
+    val targetPath :String = "/tmp/pythonparser"
+    val pythonparserFile :File = File(sourcePath)
+
+    try {
+        val targetFile :File = File(targetPath)
+        pythonparserFile.copyTo(targetFile)
+        val command = arrayOf("chmod", "+x", targetFile.getAbsolutePath())
+        val b = ProcessBuilder(*command)
+        b.directory(targetFile.parentFile)
+        b.start()
```

Comment:
I would suggest making a class for parser setup with methods like:
- putParserToTmp() 
- makeFileExecutable(file: File)
- checkSetup() // to check if there is already a pyhton parser in tmp

So before tests you will be able to check if parser is set up and set it up in case it doesnt. 

it's better to follow OOP concepts (and put everything about set up in one class!) even if it's possible to avoid creating classes :)

Reply:
You are totally right! I forgot to send a comment about it